config/webpack/webpack.config.js

```diff
@@ -152,7 +154,9 @@ const defaultWebpackConfig = (env = {}, argv = {}) => {
     resolveModules.unshift(path.resolve(rootPath, 'aggregated-translations'));
   }
 
-  const options = { rootPath, resolveModules };
+  const themeFile = aggregateThemes();
```

Comment:
We should provide an option to prevent automatic theme aggregation. It seems like this should be opt-in by default

Reply:
This functionality is opt in by default. The script will exit if there is not a `theme.config.js` file. By adding the file you are opting into theme aggregation.